allure-report-face/src/blocks/pane-header/pane-header.hbs

```diff
@@ -1,6 +1,5 @@
 <a class="pane__overlay" href="#{{baseUrl}}/{{uid}}"></a>
-<h2 class="pane__title">
-    {{title}}
+<h2 class="pane__title preformatted-text">{{title}}
```

Comment:
`preformatted-text` class also affects controls, however it shouldn't.

Create a special wrapper just for the title:

```
<h2 class="pane__title">
   <span class="pane__controls">
     // controls markup here...
   </span>
   <span class="preformatted-text">{{title}}</span>
</h2>
```

Then it will give you the expected result